examples/microwave-oven-app/microwave-oven-common/include/microwave-oven-device.h

```diff
@@ -35,13 +35,21 @@ namespace Clusters {
 /**
  * set default value for the optional attributes
  */
-constexpr uint8_t kDefaultMinPower  = 10u;
-constexpr uint8_t kDefaultMaxPower  = 100u;
-constexpr uint8_t kDefaultPowerStep = 10u;
+constexpr uint8_t kMinPower            = 20u;
+constexpr uint8_t kMaxPower            = 110u;
+constexpr uint8_t kPowerStep           = 10u;
+constexpr uint32_t kMaxCookTime        = 86400u;
```

Comment:
add unit of measurement for any time constants or variables. This should be `kMaxCookTimeSec` or `kMaxCookTimeMin` or similar.